buildconfig/pygame-stubs/cursors.pyi

```diff
@@ -0,0 +1,23 @@
+from typing import Tuple, Sequence, Optional
+
+_Bitmap = Tuple[Tuple[int, int], Tuple[int, int],
```

Comment:
This is my favourite type signature. hehe. 😻

Reply:
Hahaha my inspiration was issue 221 in python/typing (didn't put link because it looks like it gets mentioned even if I write link in other projects, don't want to overwhelm them with updates :))